src/cc_helpers.cc

```diff
@@ -505,60 +505,80 @@ void certifier::framework::cc_trust_data::clear_sensitive_data() {
   // Not necessary on most platforms.
 }
 
+//  cc_trust_data relies on the following data in the store
+//    public_policy_key
+//    public_key_algorithm
+//    symmetric_key_algorithm
+//    for attestation:
+//      service-attest-key
+//      sealing-key
+//    for authentication
+//      auth-key
+//      app-symmetric-key
+//    Some platforms require obtaining a platform rule. For them, it should
+//    be stored.  platform-rule (signed-claim)
+//  initialized cert is in the array initialized_cert with size initialized_cert_size
```

Comment:
Not sure about this new comment line.

I did not find any references to 'initialized_cert', or 'initialized_cert_size` in `certifier_framework.h`.

What are we trying to refer to here with this comment? Might need some corrections / clarifications.